akka23-conductr-client-lib/src/test/scala/com/typesafe/conductr/clientlib/akka/ControlClientSpec.scala

```diff
@@ -0,0 +1,333 @@
+      }
+    }
+
+    "handle error when unloading a bundle" in {
+      pending
+    }
+
+    // TODO: Implement test for streaming bundle events
+    "stream events of all bundles" in {
+      import cc.actorMaterializer.executionContext
+
+      // format: OFF
+      val route =
+        pathPrefix(ApiVersion / "bundles" / "events") {
+          get {
+            onComplete(Future.successful(Source.fromPublisher(ActorPublisher[ServerSentEvent](bundlesEventsPublisher())).take(10))) {
+              case Success(source) => complete(source)
+              case Failure(_) => complete(StatusCodes.TooManyRequests -> "Maximum number of bundle event publishers reached.")
+            }
+          }
+        }
+      // format: ON
+
+      pending
+    }
+
+    "handle error when streaming events of all bundles" in {
+      pending
+    }
+
+    // TODO: Implement test for retrieving events of a bundle
+    "retrieve events of a bundle" in {
+      import cc.actorMaterializer.executionContext
+
+      // format: OFF
+      val route =
+        pathPrefix(ApiVersion / "bundles") {
+          path(Segment / "events") { bundleId =>
+            get {
+              parameter("count".as[Int].?) { count =>
+                complete {
+                  HttpResponse(entity = HttpEntity(ContentTypes.`application/json`,
+                    s"""
+                       |[
+                       |  $BundleEventsAsJson
+                       |]
+                """.stripMargin)
+                  )
+                }
+              }
+            }
+          }
+        }
+      // format: ON
+
+      pending
+    }
+
+    "handle error when retrieving events of a bundle" in {
+      pending
+    }
+
+    // TODO: Implement test for retrieving logs of a bundle
+    "retrieve logs of a bundle" in {
+      pending
+    }
+
+    "handle error when retrieving logs of a bundle" in {
+      pending
+    }
+
+    // TODO: Implement test for retrieving info of all members
+    "stream information of all members" in {
+      pending
+    }
+
+    "handle error when streaming information of all members" in {
+      pending
+    }
+
+    // TODO: Implement test for retrieving info of a member
+    "retrieve information of one member" in {
+      pending
+    }
+
+    "handle error when retrieving information of one member" in {
+      pending
+    }
+
+    // TODO: Implement test for joining a member
+    "join a member" in {
+      pending
+    }
+
+    "handle error when joining a member" in {
+      pending
+    }
+
+    // TODO: Implement test for downing a member
+    "down a member" in {
+      pending
+    }
+
+    "handle error when downing a member" in {
+      pending
+    }
+
+    // TODO: Implement test for leaving a member
+    "leave a member" in {
+      pending
+    }
+
+    "handle error when leaving a member" in {
+      pending
+    }
```

Comment:
We should complete these unit tests, or at least raise an issue so that we don't forget. How do we know that they work right now?